client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -0,0 +1,189 @@
+		) }&body=${ encodeURIComponent( emailBody ) }`;
+	}
+
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	return (
+		<div className="dispute-steps">
+			<div className="dispute-steps__header">
+				{ __( 'Steps to resolve:', 'woocommercts' ) }
+			</div>
+			<ol className="dispute-steps__steps">
+				<li>
+					{
+						// TODO: add link to the issuer evidence files link. See https://github.com/Automattic/woocommerce-payments/pull/7192.
+						__(
+							"Review the claim issued by the cardholder's bank.",
+							'woocommerce-payments'
+						)
+					}
+				</li>
+				<li>
+					{ customer?.email
+						? createInterpolateElement(
+								__(
+									'<a>Email the customer</a> to address their concerns.',
+									'woocommerce-payments'
+								),
+								{
+									a: (
+										// eslint-disable-next-line jsx-a11y/anchor-has-content
+										<a
+											target="_blank"
+											rel="noopener noreferrer"
+											href={ emailLink }
+										/>
+									),
+								}
+						  )
+						: __(
+								'Email the customer to address their concerns.',
+								'woocommerce-payments'
+						  ) }
+				</li>
+				<li>
+					{
+						// TODO: link 'guidance on dispute withdrawal' to the docs when it's ready.
+						__(
+							'Provide guidance on dispute withdrawal if the customer agrees.',
+							'woocommerce-payments'
+						)
+					}
+				</li>
+				<li>
+					{ createInterpolateElement(
+						__(
+							'Challenge <challengeicon/> or accept <accepticon/> the dispute by <disputeduedate/>.',
+							'woocommerce-payments'
+						),
+						{
+							challengeicon: (
+								<ClickTooltip
+									buttonIcon={ <HelpOutlineIcon /> }
+									buttonLabel={ __(
+										'Challenge the dispute',
+										'woocommerce-payments'
+									) }
+									content={ __(
+										"Challenge the dispute if you consider the claim invalid. You'll need to provide evidence to back your claim. Keep in mind that challenging doesn't ensure a resolution in your favor.",
+										'woocommerce-payments'
+									) }
+								/>
+							),
+							accepticon: (
+								<ClickTooltip
+									buttonIcon={ <HelpOutlineIcon /> }
+									buttonLabel={ __(
+										'Accept the dispute',
+										'woocommerce-payments'
+									) }
+									content={ sprintf(
+										// Translators: %s is a formatted currency amount, eg $10.00.
+										__(
+											`Accepting this dispute will automatically close it. Your account will be charged a %s fee, and the disputed amount will be refunded to the cardholder.`,
+											'woocommerce-payments'
+										),
+										// TODO: use getDisputeFee() from https://github.com/Automattic/woocommerce-payments/pull/7118.
+										''
+									) }
+								/>
+							),
+							disputeduedate: (
+								<span className="dispute-steps__steps__response-date">
```

Comment:
I copied this from [DisputeSummaryRow's](https://github.com/Automattic/woocommerce-payments/blob/a23c791887e8776ba1f19ffdbbb337697346439f/client/payment-details/dispute-details/dispute-summary-row.tsx#L96-L119) so it looks consistent.

One doubt I have is that in the design paJDYF-9Ip-p2, when due date is still more than 7-day away, it does not show the `({n} days left to respond)` part.

<img width="860" alt="Screenshot 2023-09-14 at 07 16 13" src="https://github.com/Automattic/woocommerce-payments/assets/73803630/6c5bff3a-3b42-4d20-86e4-a0b166f722e1">

but I see in `DisputeSummaryRow`, it shows that text.